simplq/src/components/pages/Home/Nav/Burger.jsx

```diff
@@ -1,20 +1,18 @@
-/* eslint-disable jsx-a11y/click-events-have-key-events */
-/* eslint-disable jsx-a11y/no-static-element-interactions */
 import React, { useState } from 'react';
 import styles from '../../../../styles/homePage.module.scss';
-import RightNav from './RightNav';
+import LeftNav from './LeftNav';
 
 const Burger = () => {
   const [open, setOpen] = useState(false);
 
   return (
     <>
       <div className={styles['burger']} open={open} onClick={() => setOpen(!open)}>
-        <div />
-        <div />
-        <div />
+        <div open={open} onClick={() => setOpen(!open)} />
```

Comment:
If we don't require `onClick={() => setOpen(!open)}` on these divs lets remove. As the parent already has an onclick defined

Reply:
with the parent, that was not detecting the onclick function. That is why I added  onClick={() => setOpen(!open)} to the child also.